docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+        },
+      },
+      {
+        type: 'folder',
+        relations: {
+          parent: { this: {} },
+          viewer: {
+            union: {
+              child: [
+                { this: {} },
+                {
+                  tupleToUserset: {
+                    tupleset: { object: '', relation: 'parent' },
+                    computedUserset: { object: '', relation: 'viewer' },
+                  },
+                },
+              ],
+            },
+          },
+        },
+        metadata: {
+          relations: {
+            parent: { directly_related_user_types: [{ type: 'folder' }] },
+            viewer: {
+              directly_related_user_types: [{ type: 'user' }, { type: 'group', relation: 'member' }],
+            },
+          },
+        },
+      },
+      {
+        type: 'document',
+        relations: {
+          parent: { this: {} },
+          viewer: { this: {} },
+          can_read: {
+            union: {
+              child: [
+                { computedUserset: { object: '', relation: 'viewer' } },
+                {
+                  tupleToUserset: {
+                    tupleset: { object: '', relation: 'parent' },
+                    computedUserset: { object: '', relation: 'viewer' },
+                  },
+                },
+              ],
+            },
+          },
+        },
+        metadata: {
+          relations: {
+            parent: { directly_related_user_types: [{ type: 'folder' }] },
+            viewer: {
+              directly_related_user_types: [{ type: 'user' }, { type: 'group', relation: 'member' }],
+            },
+            can_read: { directly_related_user_types: [] },
+          },
+        },
+      },
+    ],
+    schema_version: '1.1',
+  }}
+/>
+
+You can identify which usersets you need to add by looking at tuples in your store that have the following structure:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'group:sales#member',
+      relation: 'viewer',
+      object: 'folder:sales',
+      _description: 'Members of the "sales" group are viewers of the "sales" folder',
+    },
+  ]}
+/>
+
+If you find a tuple like that, you’ll need to add `group#member` in the list of types allowed in the `viewer` relation of the `folder` type.
+
+## Public Access
+
+When using version 1.0, you can indicate public access to specific objects by specifying a wildcard user in a relationship to any object, e.g.:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: '*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All users are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+When you write the tuple above, all users are granted with the “viewer” relationship for the “pricing" document. You can write those kinds of tuples for any relation that is <ProductConcept section="what-are-direct-and-implied-relationships" linkName="directly assignable" /> in the model.
```

Comment:
The "directly assignable" link just takes me to the top-level "Concepts" page.

Reply:
Try to run `npm run build; npm run serve` instead of `npm run dev`

Reply:
That fixes it